src/macro.js

```diff
@@ -156,5 +160,10 @@ function getImportSources(callExpressionPath, cwd) {
     )
   }
 
-  return glob.sync(globValue, {cwd})
+  const filepaths = glob.sync(globValue, {cwd})
+  if (typeof config.transformModulePath === 'function') {
+    return filepaths.map((p) => config.transformModulePath(p))
```

Comment:
We should pass the filepath of the current file as a second argument here so folks can use it in their transform.